sdgym/synthesizers/uniform.py

```diff
@@ -1,17 +1,33 @@
 """UniformSynthesizer module."""
+import logging
+
 import numpy as np
 import pandas as pd
 from rdt.hyper_transformer import HyperTransformer
 
 from sdgym.synthesizers.base import BaselineSynthesizer
 
+LOGGER = logging.getLogger(__name__)
+
 
 class UniformSynthesizer(BaselineSynthesizer):
     """Synthesizer that samples each column using a Uniform distribution."""
 
     def _get_trained_synthesizer(self, real_data, metadata):
         hyper_transformer = HyperTransformer()
         hyper_transformer.detect_initial_config(real_data)
+        supported_sdtypes = hyper_transformer._get_supported_sdtypes()
+        config = {}
+        for column_name, column in metadata.columns.items():
+            sdtype = column['sdtype']
+            if sdtype in supported_sdtypes:
+                config[column_name] = sdtype
+            else:
+                LOGGER.info(
+                    f'Column {column} sdtype: {sdtype} is not supported, '
+                    f'defaulting to inferred type.')
```

Comment:
I think if we do this, it will drop any pii type (eg. phone_number) and use the inferred type instead, which will just be categorical. The issue says this synthesizer can use Faker for those types so I don't think we want to do that

Reply:
Updated to use the `PII` type if labeled.

Do you recommend that we infer `PII` if its one of the `SDTYPE_ANONYMIZERS` is detected (e.g. email) but **unlabeled** within the metadata